Zoom and Drag&Drop

juliephiloche opened the pull request:
Hi Brian,

Good work !
For an internal demand, I modified your extension.
I've added a zoom and drag/pan solution.

I've added a class attribute on the xml.documentElement = "svg_map",
it's more easy to retrieve it after. I've added the zoom and drag/pan
tool  at the end of the code for ease.

Thanks to merge it with your master.
Julie

brianwmunz commented:
Hi JuliePhiloche,
When I try to use the extension, it throws the following error and doesn't work:
"Uncaught SyntaxError: In strict mode code, functions can only be declared at top level or immediately within another function."
Do you get the same?

brianwmunz commented:
Hi, the extension shows up now.  Not sure how to get the zoom & pan
working?  I don't see anything in the properties.
Thanks!

On Tue, Nov 3, 2015 at 8:30 AM, Julie Philoche notifications@github.com
wrote:

> Hi Brian,
> 
> Good work !
> For an internal demand, I modified your extension.
> I've added a zoom and drag/pan solution.
> 
> I've added a class attribute on the xml.documentElement = "svg_map",
> it's more easy to retrieve it after. I've added the zoom and drag/pan
> tool at the end of the code for ease.
> 
> Thanks to merge it with your master.
> 
> ## Julie
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/brianwmunz/svgReader-QlikSense/pull/5
> Commit Summary
> - Zoom and Drag&Drop
> 
> File Changes
> - _M_ svgReader/svgReader.js
>   https://github.com/brianwmunz/svgReader-QlikSense/pull/5/files#diff-0
>   (72)
> 
> Patch Links:
> - https://github.com/brianwmunz/svgReader-QlikSense/pull/5.patch
> - https://github.com/brianwmunz/svgReader-QlikSense/pull/5.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/brianwmunz/svgReader-QlikSense/pull/5.

juliephiloche commented:
Hi,

For the moment, scroll with the mouse wheel.
I will add some buttons for mobile devices.
And also options to disable the zoom/pan and a gap choice for the zoom.

Julie

2015-11-05 15:50 GMT+01:00 Brian Munz notifications@github.com:

> Hi, the extension shows up now. Not sure how to get the zoom & pan
> working? I don't see anything in the properties.
> Thanks!
> 
> On Tue, Nov 3, 2015 at 8:30 AM, Julie Philoche notifications@github.com
> wrote:
> 
> > Hi Brian,
> > 
> > Good work !
> > For an internal demand, I modified your extension.
> > I've added a zoom and drag/pan solution.
> > 
> > I've added a class attribute on the xml.documentElement = "svg_map",
> > it's more easy to retrieve it after. I've added the zoom and drag/pan
> > tool at the end of the code for ease.
> > 
> > Thanks to merge it with your master.
> > 
> > ## Julie
> > 
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/brianwmunz/svgReader-QlikSense/pull/5
> > Commit Summary
> > - Zoom and Drag&Drop
> > 
> > File Changes
> > - _M_ svgReader/svgReader.js
> >   https://github.com/brianwmunz/svgReader-QlikSense/pull/5/files#diff-0
> >   (72)
> > 
> > Patch Links:
> > - https://github.com/brianwmunz/svgReader-QlikSense/pull/5.patch
> > - https://github.com/brianwmunz/svgReader-QlikSense/pull/5.diff
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > https://github.com/brianwmunz/svgReader-QlikSense/pull/5.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/brianwmunz/svgReader-QlikSense/pull/5#issuecomment-154081462
> .
